packages/web-components/fast-components/src/combobox/fixtures/base.html

```diff
@@ -104,85 +118,38 @@ <h3>Both Autocomplete</h3>
     <fast-option>Jodie Whittaker</fast-option>
 </fast-combobox>
 
-<h2>Disabled Combobox</h2>
-<fast-combobox disabled>
-    <fast-option>Extra Small</fast-option>
-    <fast-option>Small</fast-option>
-    <fast-option>Medium</fast-option>
-    <fast-option>Large</fast-option>
-    <fast-option>Extra Large</fast-option>
-</fast-combobox>
-
-<h2>Combobox with disabled items</h2>
-
-<h3>Combobox with alternating disabled items</h3>
-<fast-combobox id="combobox-with-every-other-disabled">
-    <fast-option>Extra Small</fast-option>
-    <fast-option disabled>Small</fast-option>
-    <fast-option>Medium</fast-option>
-    <fast-option disabled>Large</fast-option>
-    <fast-option>Extra Large</fast-option>
+<h2>Disabled combobox</h2>
+<fast-combobox id="combobox-disabled" disabled>
+    <fast-option>This option is not selectable</fast-option>
 </fast-combobox>
 
-<h3>Combobox with adjacent disabled items at start of list</h3>
-<fast-combobox id="combobox-with-adjacent-disabled-start">
-    <fast-option disabled>Extra Small</fast-option>
-    <fast-option disabled>Small</fast-option>
-    <fast-option>Medium</fast-option>
-    <fast-option>Large</fast-option>
-    <fast-option>Extra Large</fast-option>
-</fast-combobox>
-
-<h3>Combobox with adjacent disabled items in middle of list</h3>
-<fast-combobox id="combobox-with-adjacent-disabled-middle">
-    <fast-option>Extra Small</fast-option>
-    <fast-option disabled>Small</fast-option>
-    <fast-option disabled>Medium</fast-option>
-    <fast-option>Large</fast-option>
-    <fast-option>Extra Large</fast-option>
-</fast-combobox>
-
-<h3>Combobox with adjacent disabled items at end of list</h3>
-<fast-combobox id="combobox-with-adjacent-disabled-end">
-    <fast-option>Extra Small</fast-option>
-    <fast-option>Small</fast-option>
-    <fast-option>Medium</fast-option>
-    <fast-option disabled>Large</fast-option>
-    <fast-option disabled>Extra Large</fast-option>
-</fast-combobox>
-
-<h3>Combobox with all but one item disabled</h3>
-<fast-combobox id="combobox-with-all-but-one-disabled">
-    <fast-option disabled>Extra Small</fast-option>
-    <fast-option disabled>Small</fast-option>
-    <fast-option disabled>Medium</fast-option>
-    <fast-option disabled>Large</fast-option>
-    <fast-option>Extra Large</fast-option>
+<h2>Combobox with forced position above</h2>
+<fast-combobox position="above">
```

Comment:
All of the fast-combobox have an ID except the two forced position and the custom indicator. There are a few fast-select that you missed as well. They're not needed, but it seems to be a pattern.